share/gpodder/extensions/subscription_stats.py

```diff
@@ -41,11 +41,12 @@ def add_page(self, notebook, category, channels):
         scrolled = Gtk.ScrolledWindow()
         scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
 
-        store = Gtk.ListStore(str, str, str)
-        for average, name, last, paused in channels:
+        store = Gtk.ListStore(str, str, str, int)
+        for average, name, edate, paused in channels:
+            last = strftime('%x', localtime(edate))
             store.append([
                 ('%.1f' % round(average, 1)) if average > 0 else '?',
-                name if not paused else (_('%s (paused)') % name), last,
+                ('‖ ' if paused else '') + name, last, edate,
```

Comment:
The `‖` is small and hard to see, does anyone have an objection to using two heavy vertical bars `❚❚`?